rflx/model/type_.py

```diff
@@ -45,6 +48,14 @@ def package(self) -> ID:
     def type_(self) -> rty.Type:
         return rty.Undefined()
 
+    @property
+    def all_types(self) -> ty.List["Type"]:
```

Comment:
The name is not ideal, but I have no idea for a better name. You probably thought about the mathematical transitive closure when proposing `closure`, but there are many other meanings of this term, and it is not obvious to me what `closure` for a message means. I also thought about using `dependencies`, but I was unsure if it really fits, because `all_types` also returns itself. But maybe it is better to have a clearer name and accept this "anomaly".

Reply:
I have changed the name in f6151e41dd93a512ed4698bb757e460cef589cfe.

Reply:
Not perfect, but better IMHO.